src/style.css

```diff
@@ -0,0 +1,194 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  color: #000;
```

Comment:
`color` and `font-family` inherited style.
You don't need to write it to *. 
Put it to `body`.